### DISCUSSION TITLE
vnote win10下最新版本怎么找不到导出呀

### DISCUSSION THREAD

**Discussion by LiuYang-1997** [tamlok/vnote#1694] [Q&A]
导入导出按钮只有导入的没有导出的？
![image](https://user-images.githubusercontent.com/58632163/108599873-aeea9b00-73ce-11eb-8957-7b43cf59d2d9.png)

**Comment by LiuYang-1997** [comment]
是因为还没迁移过来么，下载了个2点几版本的，看到上面有导出

**Reply by tamlok** [reply]
beta.7已经支持了。

**Reply by LiuYang-1997** [reply]
哦哦哦，好的，辛苦作者大大

**Comment by tamlok** [comment]
Beta.7 now supports export.
